ufo2fontir/src/source.rs

```diff
@@ -1346,11 +1345,17 @@ mod tests {
         (source, input)
     }
 
-    fn build_static_metadata(name: &str) -> (impl Source, Context) {
+    fn flags_without_emit_ir() -> Flags {
+        Flags::default() - Flags::EMIT_IR
+    }
+
+    fn build_static_metadata(name: &str, flags: Flags) -> (impl Source, Context) {
         let _ = env_logger::builder().is_test(true).try_init();
         let (source, input) = load_designspace(name);
-        let mut flags = Flags::default();
-        flags.set(Flags::EMIT_IR, false); // we don't want to write anything down
+        assert!(
+            !flags.contains(Flags::EMIT_IR),
+            "we don't want to write anything"
```

Comment:
I don't think we need to assert flags never contains EMIT_IR here, one may actually want to call build_static_metadata with flags parameter that does contain EMIT_IR (e.g. for testing something related to writing out IR to disk). I suggest you simply remove this assert